fronts-client/src/util/CAPIUtils.ts

```diff
@@ -81,6 +82,14 @@ function getThumbnail(
 		meta.slideshow[0]
 	) {
 		return meta.slideshow[0].src;
+	} else if (meta.videoReplace) {
+		const atomProperties = extractAtomProperties(meta.replacementVideoAtom);
+		if (
+			atomProperties !== undefined &&
+			atomProperties.trailImage !== undefined
+		) {
+			return atomProperties.trailImage;
+		}
 	}
```

Comment:
I might be wrong but I think we're still unsure as to if we want the trail image of the video. It makes sense to show this on the expanded card view where the replaced video is, but I'm not sure we should be using it on the collapsed card view.

Reply:
I think both images should be the same. The collapsed view should show whatever the current media is set to. If you look at the 'Replace image' as an analogy, we show the Replacement image in both the collapsed and expanded view. I think we should do the same for video.